jobs/kola-azure.Jenkinsfile

```diff
@@ -101,6 +93,8 @@ lock(resource: "kola-azure-${params.ARCH}") {
                              file(variable: 'AZURE_KOLA_TESTS_CONFIG_AUTH',
                                   credentialsId: 'azure-kola-tests-config-auth')]) {
 
+                // to make things less wordy
+                def azure_cfg = pipecfg.clouds.azure
```

Comment:
For this file I'm a little less convinced of the value of doing this. We re-use the variables often below. Maybe we just take the variables that were defined at the top and define them here instead and leave the below variables as they were? So block right here would become:

```
def azure_testing_resource_group = pipecfg.clouds.azure.test_resource_group
def azure_testing_storage_account = pipecfg.clouds.azure.test_storage_account
def azure_testing_storage_container = pipecfg.clouds.azure.test_storage_container
```

Reply:
No strong opinion, but I do like the concept of aliasing a whole group and referencing that instead of multiple individual variables. It makes it clear they're related and referencing them is more or less the same number of characters.

If you'd prefer I switch it back, I'm fine with that.

Reply:
Yeah I think I'd prefer to switch back. This file is small and now we'll be declaring the variables closer to where we use them, which will help cut down on confusion.